domain/blockdevice/service/service.go

```diff
@@ -56,6 +60,34 @@ func NewService(st State, wf WatcherFactory, logger Logger) *Service {
 	}
 }
 
+// BlockDevices returns the block devices for a specified machine.
+func (s *Service) BlockDevices(ctx context.Context, machineId string) ([]blockdevice.BlockDevice, error) {
+	return s.st.BlockDevices(ctx, machineId)
+}
+
+// UpdateBlockDevices updates the block devices for a specified machine.
+func (s *Service) UpdateBlockDevices(ctx context.Context, machineId string, devices ...blockdevice.BlockDevice) error {
+	for i := range devices {
+		if devices[i].FilesystemType == "" {
+			devices[i].FilesystemType = "unspecified"
```

Comment:
Can we make `"unspecified"` a constant with a comment.

Reply:
done